examples/02-ops.rs

```diff
@@ -53,4 +55,16 @@ fn main() {
     let b: Tensor<Rank1<7>, f32, _> = dev.sample_normal();
     let c = a.matmul(b);
     dbg!(c.array());
+
+    // these operations are equal across devices
+    #[cfg(feature = "cuda")]
+    {
+        let cpu = Cpu::default();
+
+        let a: Tensor<Rank1<3>, f32, _> = dev.sample_normal();
+        let b: Tensor<Rank1<7>, f32, _> = dev.sample_normal();
+        let a_cpu = a.to_device(&cpu);
+        let b_cpu = b.to_device(&cpu);
+        assert_eq!(a_cpu.matmul(b_cpu).array(), a.matmul(b).array());
```

Comment:
nice!! I like that its showing to_device and also that the operations are equivalent 👍